### PR TITLE
 Replace long text offering to take an action.

### LINKED ISSUE #2958: "Take action for action phase, select one available action" is just too many words
image

There are four translations, so I think this won't be that hard to fix.

### PR DESCRIPTION
**Pull request by kberg**
Fixes #2958

### CODE REVIEW

File: src/locales/ru/ui.json [hunk 1]
```diff
@@ -132,7 +132,8 @@
     "Play project card": "Разыграть карту",
     "Sell patents": "Продать карты",
     "Fund an award": "Учредить награду",
-    "Take action for action phase, select one available action.": "Выберите одно из действий для продолжения игры",
+    "Take your first action": "Выберите одно из действий для продолжения игры",
```
Comment (alrusdi) [inline src/locales/ru/ui.json:135]: 1st should be: "Выберите первое действие для продолжения игры"
2nd:  "Выберите второе действие для продолжения игры" 
:-)

File: src/Player.ts [hunk 2]
```diff
@@ -1899,8 +1899,8 @@ export class Player implements ISerializable<SerializedPlayer> {
   // Return possible mid-game actions like play a card and fund an award, but no play prelude card.
   public getActions() {
     const action: OrOptions = new OrOptions();
-    action.title = 'Take action for action phase, select one ' +
-                      'available action.';
+    action.title = this.actionsTakenThisRound === 1 ?
```
Comment (bafolts) [inline src/Player.ts:1902]: Should this be:

```
action.title = this.actionsTakenThisRound === 0 ? 'Take your first action' : 'Take your second action';
```

I think `actionsTakenThisRound` is `0` when making a selection for first action.

Comment (kberg) [reply src/Player.ts]: You're correct. I had an unsubmitted correction I forgot to commit. Fixed.

**Review by bafolts — APPROVED** [review]

File: src/locales/ru/ui.json (hunk 1 above)
Comment (kberg) [reply src/locales/ru/ui.json]: Thank you! Fixed.

**Review by alrusdi — APPROVED** [review]

**Comment by kberg** [comment]
You know what I realized? In solo, this doesn't apply. I probably used the wrong player field.

**Comment by bafolts** [comment]
Will probably have to add a generic "Take Action" for single player games.

**Pull request merged**